Tests/MessagingPush/HttpTest.swift

```diff
@@ -33,10 +33,29 @@ open class HttpTest: UnitTest {
          */
         if let writeKey = getEnvironmentVariable("WRITE_KEY") {
             runner = UrlRequestHttpRequestRunner()
-            session = RichPushHttpClient.getCIOApiSession(
+            cioSession = RichPushHttpClient.getCIOApiSession(
                 key: writeKey,
                 userAgentHeaderValue: deviceInfo.getUserAgentHeaderValue()
             )
+            session = RichPushHttpClient.getBasicSession()
+        }
+    }
+
+    func testDownloadFileCreatesExpectedFile() {
+        if let session = session {
```

Comment:
I like this test and think we should get it running in the CI, at least in the short-term. 

Assuming you agree, could we modify `session` to be non-nil? So we don't need this `if let` condition? I would like us to feel more confident that this test will execute and having this `if let` makes me think sometimes it might not run. 

Also, what do you think about performing a 2nd HTTP request in this test function, downloading the same image URL, and expect that the 2 local file paths are unique?

Reply:
We could run it in CI, if needed. `if let` was there to ensure it doesn't run on CI but we can update that.